orocos_kdl/src/chainiksolverpos_nr.cpp

```diff
@@ -48,23 +51,49 @@ namespace KDL
             return (error = E_SIZE_MISMATCH);
 
         q_out = q_init;
+        bool degraded = false;
 
-        unsigned int i;
-        for(i=0;i<maxiter;i++){
+        for(numiter=1; numiter<=maxiter; ++numiter) {
```

Comment:
```suggestion
        for(numiter=1; numiter<=maxiter; ++numiter)
        {
```

Reply:
Wait, didn't you suggest in all other cases that the bracket go on the same line as the statement?

Reply:
No brackets should be on a newline.

Reply:
I get your confusion. I edited my previous comment.

Brackets should be on a newline.